lib/node_modules/@stdlib/math/base/special/ellipe/test/test.native.js

```diff
@@ -0,0 +1,178 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var HALF_PI = require( '@stdlib/constants/float64/half-pi' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+
+
+// FIXTURES //
+
+var mediumPositive = require( './fixtures/cpp/medium_positive.json' );
+var closeToUnity = require( './fixtures/cpp/close_to_unity.json' );
+var mediumNegative = require( './fixtures/cpp/medium_negative.json' );
+var largeNegative = require( './fixtures/cpp/large_negative.json' );
```

Comment:
I appreciate your checks on `mediumNegative` and `largeNegative`.

Reply:
Thanks for this and your suggestions, I've made the suggested changes.